revision_score.go

```diff
@@ -10,9 +10,14 @@ type RevisionScore struct {
 	baseSchema
```

Comment:
`revision-score` stream is no longer supported.

https://stream.wikimedia.org/?doc#/streams

Let's remove `revision_score.go`, it's unit test, from client.go, options.go, builder.go and references from their unittest.

Reply:
Done.